src/app/client/projects/desktop/src/app/modules/offline/components/offline-report-issues/offline-report-issues.component.spec.ts

```diff
@@ -1,25 +1,91 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
 import { OfflineReportIssuesComponent } from './offline-report-issues.component';
+import { ConfigService, ResourceService, BrowserCacheTtlService, ToasterService, SharedModule } from '@sunbird/shared';
+import { of as observableOf, throwError } from 'rxjs';
+import { SuiModalModule } from 'ng2-semantic-ui';
+import { CacheService } from 'ng2-cache-service';
+import { OfflineReportIssuesService } from '../../services/offline-report-issues/offline-report-issues.service';
```

Comment:
import { OfflineReportIssuesService } from '../../services';
if this is not working check if OfflineReportIssuesService exported from service folder's index.ts